src/controllers/confirmation.controller.ts

```diff
@@ -51,25 +51,31 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
       });
   }
 
-  const token = req.chSession.accessToken() as string;
-  let apiResponseData: any;
-  try {
-    // TODO  LFA-1406 Add isSubmitted flag to prevent this being sent twice
+  const isSubmitted: boolean = getPromiseToFileSessionValue(req.chSession, ALREADY_SUBMITTED);
+  if (!isSubmitted) {
+    const token = req.chSession.accessToken() as string;
+    let apiResponseData: any;
+    try {
 
-    const axiosResponse: AxiosResponse = await callPromiseToFileAPI(companyProfile.companyNumber,
-        token, isStillRequired);
+        const axiosResponse: AxiosResponse = await callPromiseToFileAPI(companyProfile.companyNumber,
```

Comment:
Looks like you've switched to 4 spaces for a tab, but 2 is the standard.

Reply:
Changed to 2 spaces.